assets/js/googlesitekit/modules/index.js

```diff
@@ -30,12 +30,59 @@ import './datastore';
 
 const Modules = {
 	createModuleStore,
+	/**
+	 * Activates Module
+	 *
+	 * Activates a module in the modules store based on the provided args.slug parameter.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param {Object} [args]      Module arguments.
+	 * @param {string} [args.slug] Slug of the module to activate.
```

Comment:
Even though the function passes through all arguments to the action creator, we should document the arguments as if they were passed individually (this should arguably be changed in the function signature as well but that's outside of the scope here).

```suggestion
	 * @param {string} slug Slug of the module to activate.
```

Also, `slug` is not an optional parameter to the action but it is documented as an optional argument here using the bracket syntax.

Please do the same for the others here as well, (documenting individually – no `args.` prefix – and matching optional – only `registerModule` has optional `settings`)

Reply:
I had to pass args.slug to satisfy ESLint.

Reply:
Otherwise updated.